start.sh

```diff
@@ -2,6 +2,7 @@
 set -e
 
 INPUT_FORCE=${INPUT_FORCE:-false}
+INPUT_FORCE_WITH_LEASE=${INPUT_FORCE_WITH_LEASE:-false}
```

Comment:
We can reduce redundant default declarations and provide them in the actions.yml

Reply:
Same as above.